naming/src/main/java/org/jboss/as/naming/subsystem/NamingBindingResourceDefinition.java

```diff
@@ -117,22 +125,70 @@ public void registerAttributes(ManagementResourceRegistration resourceRegistrati
     }
 
     @Override
+    public void registerOperations(ManagementResourceRegistration resourceRegistration) {
+        super.registerOperations(resourceRegistration);
+        SimpleOperationDefinitionBuilder builder = new SimpleOperationDefinitionBuilder(NamingSubsystemModel.REBIND, getResourceDescriptionResolver())
+                .addParameter(BINDING_TYPE)
+                .addParameter(TYPE)
+                .addParameter(VALUE)
+                .addParameter(CLASS)
+                .addParameter(MODULE)
+                .addParameter(LOOKUP)
+                .addParameter(ENVIRONMENT);
+        resourceRegistration.registerOperationHandler(builder.build(), new OperationStepHandler() {
+            @Override
+            public void execute(OperationContext context, ModelNode operation) throws OperationFailedException {
+                context.addStep(new OperationStepHandler() {
+                    @Override
+                    public void execute(OperationContext context, ModelNode operation) throws OperationFailedException {
+
+                        validateResourceModel(operation, false);
+                        Resource resource = context.readResourceForUpdate(PathAddress.EMPTY_ADDRESS);
+                        ModelNode model = resource.getModel();
+                        for (AttributeDefinition attr : ATTRIBUTES) {
+                            attr.validateAndSet(operation, model);
+                        }
+
+                        context.addStep(new OperationStepHandler() {
+                            @Override
+                            public void execute(OperationContext context, ModelNode operation) throws OperationFailedException {
+                                final String name = context.getCurrentAddressValue();
+                                final ContextNames.BindInfo bindInfo = ContextNames.bindInfoFor(name);
+                                ServiceController<ManagedReferenceFactory> service = (ServiceController<ManagedReferenceFactory>) context.getServiceRegistry(false).getService(bindInfo.getBinderServiceName());
+                                if (service == null) {
+                                    context.reloadRequired();
+                                    return;
+                                }
+                                NamingBindingAdd.INSTANCE.doRebind(context, operation, (BinderService) service.getService());
+                            }
+                        }, OperationContext.Stage.RUNTIME);
+                    }
+                }, OperationContext.Stage.MODEL);
+            }
+        });
+    }
+
+    @Override
     public List<AccessConstraintDefinition> getAccessConstraints() {
         return ACCESS_CONSTRAINTS;
     }
 
+    public void registerTransformers_2_1(ResourceTransformationDescriptionBuilder builder) {
```

Comment:
I'd call this 2_0, since it is the transformers to 2.0. 2.1.0 is the current version which will change in the future